scss/cayman.scss

```diff
@@ -1,6 +1,21 @@
+// Breakpoints
 $large-breakpoint: 64em;
 $medium-breakpoint: 42em;
 
+// Sass variables
+$brand-color: #159957;
+$secondary-brand-color: #155799;
+$light-color: #FFFFFF;
+$body-text-color: #606c71;
+$body-link-color: #1e6bb8;
+$button-text-color: $light-color;
+$code-bg-color: #f3f6fa;
+$code-text-color: #567482;
+$gray-blue: #dce6f0;
+$blockquote-text-color: #819198;
+$table-border-color: #e9ebec;
+$ruler-color: #eff0f1;
```

Comment:
Let's go with `$hr-border-color` here even though it's not used for just `<hr>`s.